src/framebuffers.h

```diff
@@ -36,11 +36,11 @@ typedef struct {
  * This type could reasonably be called activeCharacterPalette or similar
  *
  * Appears to be a combination of 2 different palettes:
- *   kart_palette appears to be the palette for all non-wheel elements of the kart (including the driver).
- *   wheel_palette then is the palette for the wheels
- *
- * kart_palette does not appear to be used 100% of the time, more testing is needed to figure out what's going on
- * wheel_palette changes as you drive around while kart_palette is constant
+ *   kart_palette contains the pallete for all non-wheel elements of the kart (including the driver).
+ *   wheel_palette contains the palette for the wheels.
+ * 
+ * kart palette sets a defined palette based on the character.
+ * while wheels pallete sets a dynamic palette as your drive around with the kart.
```

Comment:
~~your~~ you
`wheel_palette sets a dynamic palette as you drive around with the kart.`